internal/logger/telemetry_store_watermill.go

```diff
@@ -51,7 +51,7 @@ func (m *TelemetryStoreWMMiddleware) TelemetryStoreMiddleware(h message.HandlerF
 		}
 
 		ts := &TelemetryStore{
-			Project:  inf.ProjectID.String(),
+			Project:  inf.ProjectID,
 			Provider: inf.Provider,
 			Resource: resourceFromEntity(typ, ent),
```

Comment:
Should this fill in the same `repository`, `artifact`, `profile`, etc fields?  (Or have those use this format?)

Ideally, we should have the same resource keys in the logs for both RPCs that update a resource, and rule evaluations that use that resource.  That will make it easier to ask questions like "how many newly added resources have some action taken on them immediately (rule eval with at least one negative result within 30 minutes)?"

Reply:
I'll see what else we can extract from the entity message here so we can put them in the TelemetryStore object 👍 

Depending on the message some of the fields might be missing, but that should be expected.

Reply:
Yes, partially-filled structs is expected.  What would be nice would be if fields like `project` (across minder) and `login_sha` (across Minder RPC and Trusty RPC) could be consistent.  Right now, I think Ozz used my `Resource string` formulation, and you've added separate fields for each resource type.  Both are fine, but I'd rather un-tangle the two at the source than later on in Tableau.